core/src/main/java/io/grpc/internal/AbstractServerImplBuilder.java

```diff
@@ -175,11 +176,15 @@ public ServerImpl build() {
     StatsContextFactory statsFactory =
         this.statsFactory != null ? this.statsFactory : Stats.getStatsContextFactory();
     if (statsFactory != null) {
-      CensusStreamTracerModule census =
-          new CensusStreamTracerModule(statsFactory, GrpcUtil.STOPWATCH_SUPPLIER);
-      tracerFactories.add(census.getServerTracerFactory());
+      CensusStatsModule censusStats =
+          new CensusStatsModule(statsFactory, GrpcUtil.STOPWATCH_SUPPLIER);
+      tracerFactories.add(censusStats.getServerTracerFactory());
     }
+    CensusTracingModule censusTracing =
+        new CensusTracingModule(Tracing.getTracer(), Tracing.getBinaryPropagationHandler());
```

Comment:
Hmm... so we'll send headers all the time? Do we want to flag-protect this? We can use a static field and just enable it when we want to test.